lms/static/sass/_build-course.scss

```diff
@@ -60,8 +60,7 @@
 @import "course/ccx_coach/dashboard";
 
 // discussion
-@import "course/discussion/form-wmd-toolbar";
-@import "course/discussion/form";
+@import 'build-discussion-editor';
```

Comment:
@andy-armstrong I looked into this. Here is what I found:

* `course/discussion/form` targets `.wmd-prompt-dialog`
* `.wmd-prompt-dialog` comes from `Markdown.Editor.js`
* `Markdown.Editor.js` is only referenced in `discussion_vendor_js`
* `course/discussion/form-wmd-toolbar` only targets elements that belong to discussion-related components

Based on this,  `course/discussion/form-wmd-toolbar` and `course/discussion/form` are not needed here. We'll need to make sure they are loaded on the "Teams" tab though; the template for that tab [imports](https://github.com/open-craft/edx-platform/blob/itsjeyd/inline-discussion-assets/lms/djangoapps/teams/templates/teams/teams.html#L17) `lms-course.css`, so it's relying on the styles defined here. The approach you describe [here](https://github.com/edx/edx-platform/pull/13483/files#r80835975) should work.